README.md

```diff
@@ -25,12 +25,12 @@ The Delphi registry requires a docker host to deploy containers to. The followin
 * The Delphi WebApi ( ```delphi-webapi:1.0.0-SNAPSHOT``` )
 * The Delphi WebApp ( ```delphi-webapp:1.0.0-SNAPSHOT``` )
 
-To obtain these images, checkout the respective repositories ([here](https://github.com/delphi-hub/delphi-crawler), [here](https://github.com/delphi-hub/delphi-webapi) and [here](https://github.com/delphi-hub/delphi-webapp)) and execute the command 
+To obtain these images, please checkout the Delphi registry repository and execute the command 
 
```

Comment:
Please also keep the old content of the readme and address both Windows and Linux. I thought of something like this:
"For Linux, please execute the setup script located at [...], for windows <old content of readme here>."

Reply:
Thanks for the review. I have made the requested changes. Could you please review and let me know for any other changes.